packages/apps/human-app/frontend/src/pages/worker/jobs/jobs.page.tsx

```diff
@@ -29,13 +30,15 @@ function generateTabA11yProps(index: number) {
 export function JobsPage() {
   const { isDarkMode } = useColorMode();
   const { data, isError, isPending, error } = useGetOracles();
+  const { data: uiConfigData, isPending: isPendingUiConfig } = useGetUiConfig();
```

Comment:
If config loading fails - we show some default value and no name in network column
<img width="304" alt="Screenshot 2024-11-28 at 15 34 56" src="https://github.com/user-attachments/assets/a29f842e-554d-43c7-99ad-18619fa17694">

Let's show no records and error alert instead